cmd/serve-config.go

```diff
@@ -134,8 +138,15 @@ func serveConfigRun(cmd *cobra.Command, args []string) error {
 		AWSWorkerRole: awsEnv.READER_ROLE_NAME,
 		AWSOrgRoles:   awsEnv.ORG_ROLE_ARNS,
 		Concurrency:   concurrency,
+		SkipAccounts:  sets.StringSet{},
 	}
 
+	for _, skipAcct := range skipAccountList {
```

Comment:
I think instead you can do `configGenerationParams.SkipAccounts.Add(skipAccountList...)` to add them all